common/app/http/Filters.scala

```diff
@@ -71,6 +72,23 @@ class SurrogateKeyFilter(implicit val mat: Materializer, executionContext: Execu
   }
 }
 
+class ExperimentsFilter(implicit val mat: Materializer, executionContext: ExecutionContext) extends Filter {
+
+  override def apply(nextFilter: (RequestHeader) => Future[Result])(request: RequestHeader): Future[Result] = {
```

Comment:
could split this function for readability

Reply:
done https://github.com/guardian/frontend/pull/18320/files#diff-63195b61cf92b9a86ecd3f030f7da391R84